packages/frontend/src/lib/pnl.ts

```diff
@@ -7,7 +7,14 @@ type ShortPnLParams = {
 }
 
 export function calcUnrealizedPnl({ wethAmount, buyQuote, ethPrice }: ShortPnLParams) {
-  if (wethAmount.isEqualTo(0) || buyQuote.isEqualTo(0) || ethPrice.isEqualTo(0)) {
+  if (
+    wethAmount.isEqualTo(0) ||
+    !wethAmount.isFinite() ||
+    buyQuote.isEqualTo(0) ||
+    !buyQuote.isFinite() ||
+    ethPrice.isEqualTo(0) ||
+    !ethPrice.isFinite()
```

Comment:
Just wondering would the ethprice be finite?

Reply:
it may never be but .isFinite covers for the number not being NaN too

Reply:
Maybe using isNaN would be more accurate?

Reply:
I think .isFinite covers more cases

Reply:
kk gotcha